### ISSUE TITLE
SLA: Ignore objects that have not yet been checked in a given time period

### ISSUE THREAD

**Issue by yhabteab** [Icinga/icingadb#558]
See here a fix for the IDO reports.
https://github.com/Icinga/icingaweb2-module-idoreports/pull/51

**Comment by julianbrost** [comment]
With "not yet been checked", do you mean objects that were added later, like if I generate a report for last year, what should it say about a host that was only added this year?

**Comment by yhabteab** [comment]
Yes, in my demo, I have added the hosts on Monday (i.e Icinga2/DB setting up freshly). Now when I generate hosts report for last month, Icinga DB reports are all at `100%`, where the monitoring modules are only at `0%`.

**Comment by Al2Klimov** [comment]
Does https://github.com/Icinga/icingadb/pull/560#issuecomment-1434392316 mean that this issue should be moved to db-web for being closed by Icinga/icingadb-web#710 or even closed completely?
